src/zos_tso/zowe/zos_tso_for_zowe_sdk/tso.py

```diff
@@ -293,10 +293,10 @@ def __get_tso_data(self, session_key: str) -> dict:
 
         Returns
         -------
-        dict
+        dict[str, Any]
             A json response of the operation result
         """
         custom_args = self._create_custom_request_arguments()
         custom_args["url"] = "{}/{}".format(self._request_endpoint, session_key)
-        command_output = self.request_handler.perform_request("GET", custom_args)["tsoData"]
+        command_output:dict[str,Any] = self.request_handler.perform_request("GET", custom_args)["tsoData"]
```

Comment:
(nitpick) the formatting here seems inconsistent w/ other changes in this branch